src/server/game/Battlefield/Zones/BattlefieldWG.h

```diff
@@ -408,6 +408,7 @@ class TC_GAME_API BattlefieldWG : public Battlefield
 
         uint32 m_tenacityStack;
         uint32 m_saveTimer;
+        uint8 m_buffedTeam;
```

Comment:
@jackpoz, You Correct it's need more work and debugging.
Can you please check my last commit.